commit ed0c18ae4aae48953a96b1607266337c9f110381
Author: Romaric Jodin <rjodin@google.com>
Date:   Tue Oct 14 11:33:51 2025 +0200

    meson: add depend_files for gl_enums.py
    
    gl_enums.py depends on (at least):
    - license.py
    - gl_XML.py
    - some XML files
    
    Adding glapi_xml_py_deps to make sure that ninja-to-soong can produce
    valid blueprint for Android
    
    Ref #14072
    
    Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/37865>

diff --git a/src/mapi/glapi/gen/meson.build b/src/mapi/glapi/gen/meson.build
index d605bd70fad..136ea56d85e 100644
--- a/src/mapi/glapi/gen/meson.build
+++ b/src/mapi/glapi/gen/meson.build
@@ -24,6 +24,7 @@ main_enums_c = custom_target(
   input : ['gl_enums.py', files('../registry/gl.xml')],
   output : 'enums.c',
   command : [prog_python, '@INPUT0@', '-f', '@INPUT1@'],
+  depend_files : glapi_xml_py_deps,
   capture : true,
 )
 
